frontend/src/metabase/containers/SaveQuestionModal.unit.spec.tsx

```diff
@@ -387,6 +387,52 @@ describe("SaveQuestionModal", () => {
         expect(screen.getByRole("button", { name: "Save" })).toBeDisabled();
       });
     });
+
+    it("should show humanized validation message for saving a native query question", async () => {
+      await setup(
+        new Question(
+          {
+            dataset_query: {
+              type: "native",
+              database: ORDERS_ID,
+              native: {
+                query: "select * from orders",
+              },
+              display: "table",
+            },
+          },
+          metadata,
+        ),
+      );
+
+      const inputEl = screen.getByLabelText("Name");
+      userEvent.click(inputEl);
+      userEvent.tab();
+
+      // eslint-disable-next-line testing-library/no-node-access
+      const labelEl = document.querySelector(`label[for="${inputEl.id}"]`);
```

Comment:
maybe just `screen.getByText`? I think given the text (it contains the name of the field) it should be fine